plugins/cost-insights/src/types/Duration.ts

```diff
@@ -29,74 +26,3 @@ export enum Duration {
   P1M = 'P1M',
   P3M = 'P3M',
 }
-
-export const DEFAULT_DATE_FORMAT = 'YYYY-MM-DD';
```

Comment:
Is there a reason to move this out to utils ?

Reply:
Prior impl made these non-public. I'll roll back these changes and cut a separate PR